src/main/java/EchoUtils.java

```diff
@@ -0,0 +1,7 @@
+public class EchoUtils {
+    public static void echo(String input) {
```

Comment:
ooh looks quite clean, but do you think you need a whole class for a single function